src/exploratory.py

```diff
@@ -39,13 +39,15 @@ def __init__(
         event_name_col: str = "event_name",
         value_col: str = "purchase_value",
         segment_feature_cols: List[str] = None,
+        decimals: int = 1,
```

Comment:
nit: could decimalPlaces or roundingPrecision be a better name for this?

Reply:
great point, changing it now